Makefile.template

```diff
@@ -159,7 +159,7 @@ prepare_nncc_internal: $(WORKSPACE)
 ifeq (,$(findstring android,$(TARGET_OS)))
 	EXTERNAL_FLATC=$(BUILDTOOL_PATH)/bin/flatc ./nncc configure -DBUILD_GTEST=OFF -DENABLE_TEST=OFF -DEXTERNALS_BUILD_THREADS=$(NPROCS) -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) \
 		-DCMAKE_INSTALL_PREFIX=$(OVERLAY_FOLDER) \
-		-DBUILD_WHITELIST="luci;foder;pepper-csv2vec;loco;locop;logo;logo-core;mio-circle08;luci-compute;oops;hermes;hermes-std;angkor;pp;pepper-strcast;pepper-str" \
+		-DBUILD_WHITELIST="luci;foder;pepper-csv2vec;loco;locop;logo;logo-core;mio-circle09;luci-compute;oops;hermes;hermes-std;angkor;pp;pepper-strcast;pepper-str" \
```

Comment:
To reduce laborious changes, we've decided not to change names.
plz refer #13665

Reply:
> To reduce laborious changes, we've decided not to change names. plz refer #13665

Did I understand correctly - we use mio-circle08 and replace there schema file path?

Reply:
> we use mio-circle08

yes

>  replace schema file path in CmakeLists.txt?

maybe replace means (1) add `res/CircleSchema/0.9/circle_schema.fbs` (2) edit `mio-circle08` to use this new file.